Services/Common/GroupService.cs

```diff
@@ -34,6 +36,8 @@ private async Task ForceLoadGroup(string slug)
         if (_context != null)
             await _context.DisposeAsync();
 
+        CurrentPerson = null;
```

Comment:
This should only be done if the slug changes.
If not, it should automagically find the new person (e.g. by saving the ID, instance of the reference)

This causes wildly unexpected behavior (person is suddenly null).
Currently, this causes the screen flickering if somebody adds something to their cart